redaxo/src/core/lib/console/package/package.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ */
+abstract class rex_command_package extends rex_console_command
```

Comment:
Bin leider bissl spät zu der Party: ich finde diese abstraktion an dieser stelle hat keinen/wenig mehrwert und nimmt uns sogar Flexibilität weg in den subklassen.

Ich persönlich fand es mit „doppeltem code“ besser

Reply:
@gharlan Meinungen?

Reply:
@bloep wie ist dein Gefühl?

Reply:
Wenn ihr es beide so besser findet können wir es gerne so lassen..

Reply:
Ich kann auch gut mit beiden leben.
War bei mir so ein Reflex: "Gleicher Code überall, überall wollen wir jetzt den Exit-Code ändern, wäre es da nicht besser, das gleich auszulagern".
Aber du hast auch Recht, Abstrahieren muss nicht immer richtig/gut sein.

Ich kann mich nicht entscheiden, was ich nun besser finde!
Warten wir mal, was @bloep sagt.

Reply:
Natürlich ist Abstraktion nicht immer gut. Ich sehe nur hier Code-Dopplungen, die sich durch Abstraktion wunderbar minimieren lassen und die Command-Klassen schlanker gestalten.
Aber mir soll es im Endeffekt gleich sein.

Reply:
>ich finde diese abstraktion an dieser stelle hat keinen/wenig mehrwert...


Noch einmal über @staabm's Aussage nachgedacht. Die Abstraktion hat bis auf die kleinen Vermeidungen von Code-Dopplungen wenig Mehrwert.
Fazit: Würde mich eher gegen eine Abstraktionsklasse entscheiden.

Wenn von eurer Seite keine weiteren Einwände sind, würde ich die Abstraktion wieder entfernen.

Reply:
Von Markus wird es keine Einwände geben. Ich kann mich nicht entscheiden und enthalte mich. Daher gerne wieder entfernen und sorry für das Hin und Her!